osbs/build/user_params.py

```diff
@@ -148,7 +129,6 @@ def __init__(self, build_json_dir=None):
             "arrangement_version",
             allow_none=True,
             default=REACTOR_CONFIG_ARRANGEMENT_VERSION)
-        self.build_from = BuildParam('build_from')
```

Comment:
build_from should stay, but instead build_image and build_imagestream should be removed/deprecated also in conf.py

https://github.com/containerbuildsystem/osbs-client/blob/master/osbs/build/user_params.py#L219

Reply:
build_from is the argument, but the internal data structures are build_image and build_imagestream.  build_from gets split into a source_type and either a build_image or a build_imagestream.

Let's finish deprecating build_image and build_imagestream as arguments in a separate PR.

Reply:
ahhh ok right we don't need self.build_from 
set_param should get it and split it to build_image or build_imagestream
and that is now getting all 3 options, and should get only build_from

but sure we can do it in another PR